openlibrary/catalog/merge/names.py

```diff
@@ -127,16 +127,17 @@ def match_marc_name(marc1, marc2, last_name_only_ok):
     m1 = re_marc_name.match(marc1)
     m2 = re_marc_name.match(marc2)
     if not m1:
-        if m2 and marc1_normalized == normalize(m2.group(1)):
+        if m2 and m1_normalized == normalize(m2.group(1)):
             return last_name_only_ok
         else:
             return False
     if not m2:
-        if marc2_normalized == normalize(m1.group(1)):
+        if m2_normalized == normalize(m1.group(1)):
             return last_name_only_ok
         else:
             return False
-    if marc1_normalized == normalize(m2.group(2) + ' ' + m2.group(1)) or marc2_normalized == normalize(m1.group(2) + ' ' + m1.group(1)):
+    if (m1_normalized == normalize(m2.group(2) + ' ' + m2.group(1))) or \
```

Comment:
NFBS: (no backslashes)  They are redundant inside of (), [], {} and the real problem with them in Python is that adding one just whitespace character to the right of the backslash will break your script on a change that is invisible to the reader.

Reply:
Well noted - doing it now!